proposals/0018-clique-service.md

```diff
@@ -0,0 +1,30 @@
+---
+simd: '0018'
+title: Clique Service
+authors:
+  - Maximilian Schneider (Mango)
+category: Networking
```

Comment:
Category for this would be `Standard` with type being `Networking` as defined in the [SIMD template](https://github.com/solana-foundation/solana-improvement-documents/blob/main/XXXX-template.md)

Reply:
I went with optional as this proposal relates to an optional / experimental feature of the protocol